forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -449,377 +249,264 @@ const Edit = React.memo(() => {
-                        type="radio"
-                        name="display"
-                        id="form-radio-form"
-                        value="form"
-                        checked={form.display === "form"}
-                        onChange={(event) => handleChange("display", event)}
-                      />
-                      <label className="form-check-label fw-light" htmlFor="form-radio-form">
-                        {t("Form")}
-                      </label>
+            </Card.Body>
+          </Card>
+          <div className="d-flex mb-3">
+            <div className={`wraper form-wraper ${showLayout ? 'visible' : ''}`}>
+              <Card>
+                <Card.Header>
+                  <div className="d-flex justify-content-between align-items-center" style={{ width: "100%" }}>
+                    <div className="d-flex align-items-center justify-content-between">
+                      <div className="mx-2 builder-header-text">Layout</div>
+                      <div>
+                        <CustomButton
+                          variant="secondary"
+                          size="md"
+                          icon={<HistoryIcon />}
+                          label={<Translation>{(t) => t("History")}</Translation>}
+                          onClick={handleHistory}
+                          dataTestid={"cancelBtndataTestid"}
+                          ariaLabel={"cancelBtnariaLabel"}
+                        />
+                        <CustomButton
+                          variant="secondary"
+                          size="md"
+                          className="mx-2"
+                          icon={<PreviewIcon />}
+                          label={<Translation>{(t) => t("Preview")}</Translation>}
+                          onClick={handlePreview}
+                          dataTestid={"cancelBtndataTestid"}
+                          ariaLabel={"cancelBtnariaLabel"}
+                        />
+                      </div>
                     </div>
-                    <div className="form-check form-check-inline">
-                      <input
-                        data-testid="form-edit-wizard-display"
-                        className="form-check-input"
-                        type="radio"
-                        name="display"
-                        id="form-radio-wizard"
-                        value="wizard"
-                        checked={form.display === "wizard"}
-                        onChange={(event) => handleChange("display", event)}
+                    <div>
+                      <CustomButton
+                        variant="primary"
+                        size="md"
+                        className="mx-2"
+                        label={<Translation>{(t) => t("Save Layout")}</Translation>}
+                        onClick={saveLayout}
+                        dataTestid={"save-form-layout"}
+                        ariaLabel={"Save Form Layout"}
```

Comment:
give translation tag to the arialabel text (ex : {{(t) => t("Discard Changes")}} )